maintainers/maintainer-list.nix

```diff
@@ -2817,6 +2817,12 @@
     githubId = 541748;
     name = "Felipe Espinoza";
   };
+  fehnomenal = {
+    email = "fehnomenal@fehn.systems";
+    github = "fehnomenal";
+    githubId = 9959940;
+    name = "Andreas Fehn";
```

Comment:
Please put this into a separate commit with the message: ``maintainers: add fehnomenal``.

Reply:
Done.